cmd/utils/flags.go

```diff
@@ -1618,6 +1645,8 @@ func SetBuilderConfig(ctx *cli.Context, cfg *builder.Config) {
 	cfg.RemoteRelayEndpoint = ctx.String(BuilderRemoteRelayEndpoint.Name)
 	cfg.SecondaryRemoteRelayEndpoints = strings.Split(ctx.String(BuilderSecondaryRemoteRelayEndpoints.Name), ",")
 	cfg.ValidationBlocklist = ctx.String(BuilderBlockValidationBlacklistSourceFilePath.Name)
+	cfg.BuilderRateLimitDuration = ctx.String(BuilderRateLimitDuration.Name)
+	cfg.BuilderRateLimitMaxBurst = ctx.Int(BuilderRateLimitMaxBurst.Name)
```

Comment:
what about 500ms that is used in 
`runRetryLoop(ctx, 500*time.Millisecond, func() {`
I remember we were talking about that too and this one might actually make a difference in builder performance